sdk/pack/variables/map_test.go

```diff
@@ -0,0 +1,17 @@
+package variables
+
+import (
+	"testing"
+
+	"github.com/shoenig/test/must"
+)
+
+func TestIDKeyedVarMap_Variables(t *testing.T) {
+	// TODO: needs to be implemented
```

Comment:
Would be nice to have those indeed. I'm fine with these tests coming in a separate PR though.